.gitignore

```diff
@@ -98,6 +98,11 @@ apps/*/dist
 apps/prairielearn/public/build
 packages/*/dist
 
+## Coverage reports
+coverage/
+apps/*/coverage
+packages/*/coverage
+
```

Comment:
Should these be added to .dockerignore and .eslintignore?

Reply:
It can't hurt! Done in f736f70; I made a few more opportunistic changes to our ignorefiles while I was there.